docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -33,8 +33,8 @@ update.
 [[ilm-phase-transitions]]
 === Phase transitions
 
-{ilm-init} moves indices through the lifecycle according to their age.
-To control the timing of these transitions, you set a _minimum age_ for each phase. For an index to
+{ilm-init} moves indices through the lifecycle according to their `age` (time since rollover fallback index creation).
```

Comment:
```suggestion
{ilm-init} moves indices through the lifecycle according to their `age` (time since rollover, falling back to index creation if it has not yet rolled over).
```